src/runtime_src/core/edge/drm/zocl/zocl_cu_xgq.c

```diff
@@ -451,12 +463,12 @@ zcu_xgq_cmd_start_cuidx(struct zocl_cu_xgq *zcu_xgq, struct xgq_cmd_sq_hdr *cmd)
 	int bit_idx;
 
 #if 0
-	zcu_xgq_cmd_complete(ZCU_XGQ2PDEV(zcu_xgq), cmd, 0);
+	zcu_xgq_cmd_complete(ZCU_XGQ2PDEV(zcu_xgq), cmd, 0, 0);
```

Comment:
This new value is a status value.  Please indicate a successful value.
```suggestion
	zcu_xgq_cmd_complete(ZCU_XGQ2PDEV(zcu_xgq), cmd, 0, KDS_COMPLETED);
```